components/nav.js

```diff
@@ -29,6 +29,46 @@ const Navbar = glamorous.nav(({top, theme: {mediaQueries}}) => ({
     flexDirection: 'column',
   },
 }))
+// Abstract input primitives to a higher level at a later date
+const inputBase = (props, {colors}) => ({
+  width: 120,
+  borderRadius: 50,
+  paddingTop: 4,
+  paddingBottom: 4,
+  paddingLeft: 30,
+  paddingRight: 20,
+  backgroundImage: 'url(/static/images/search.svg)',
```

Comment:
Alright @kentcdodds - you can decide if it's still worth the server response optimization. :) I won't be able to have a go at that until this evening most likely.